packages/@o3r/core/package.json

```diff
@@ -129,5 +129,11 @@
   "ng-update": {
     "migrations": "./dist/migration.json",
     "packageGroup": []
+  },
+  "exports": {
```

Comment:
Not sure it is a good idea, this package is flagged as module type (that why it contains only .mjs file except these ones to be run by the ngCLI).
Instead of exposing the helpers here, you may need to move them to @o3r/schematics